src/prevector.h

```diff
@@ -252,7 +252,7 @@ class prevector {
         resize(n);
     }
 
-    explicit prevector(size_type n, const T& val = T()) : _size(0) {
+    explicit prevector(size_type n, const T& val) : _size(0) {
```

Comment:
The `explicit` keyword is also redundant for constructions with >1 argument.

Reply:
FYI: looks like this will change post C++11:
https://en.cppreference.com/w/cpp/language/explicit
https://en.cppreference.com/w/cpp/language/converting_constructor

Reply:
Neat!

It's not really relevant here, but I'll expand on @Empact's comment because it's something I didn't know, maybe others will benefit as well.

Post c++11:
```c++
    using ScriptType = prevector<28, unsigned char>;
    ScriptType vec{1, 255};   // always works
    vec = ScriptType{1, 255}; // always works
    vec = {1, 255};           // only works if prevector's ctor is not explicit.
```
caveat: the 3rd example above does not actually work because of prevector's templated iterator ctor, but that's unrelated.